dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.json

```diff
@@ -0,0 +1,1060 @@
+{
+ "title": "converted from jotform",
+ "pages": [
+  {
+   "name": "page 1",
+   "elements": [
+    {
+     "type": "text",
+     "name": "userId",
```

Comment:
do you need userId here? It will be stored in the table but we I don't think we'll need a variable for it

Reply:
don't need as fields. Right now they are visual reminders for the reader that they will exist.  will remove before launch.